src/VisualStudio/Core/Def/KeybindingReset/KeybindingResetDetector.cs

```diff
@@ -91,13 +92,13 @@ internal sealed class KeybindingResetDetector : ForegroundThreadAffinitizedObjec
         public KeybindingResetDetector(
             IThreadingContext threadingContext,
             IGlobalOptionService globalOptions,
-            VisualStudioInfoBar infoBarService,
```

Comment:
i don't get it.  shouldn't this have broken mef?  how was this type getting instantiated?  and how did this not break anything else importing KeybindingResetDetector?